jui-core/index.d.ts

```diff
@@ -7,6 +7,10 @@
 
 declare var jui: JuiStatic;
 
+declare type ClassFunction = (() => any);
```

Comment:
Use `declare interface ClassFunction { new(): any; }`.
Or, just replace the use of `ClassFunction` with `{ new(): any; }`.